### DISCUSSION TITLE
BUG TRAIN CANT MOVE (CANT DO ANYTHING)

### DISCUSSION THREAD

**Discussion by mricebear** [voxeltycoon/issues#900] [Troubleshooting] [answered]
![20210916202142_1](https://user-images.githubusercontent.com/87141731/133620077-7c073422-ec79-4f6b-b3d6-4fe27b4180d7.jpg)

I'm tired this always happen and i need to save and load everytime this occur,

Pls fix this

**Comment by DanielBarreleiro** [comment]
> I'm tired this always happen and i need to save and load everytime this occur,
> 
> Pls fix this

Are you using any mods?

Try uploading here the file it opens when clicking the red button

**Comment by victorleblais** [comment, marked as answer]
You should provide the logs so that the devs can start finding out what's wrong.

Here is were logs are: https://github.com/voxeltycoon/issues/discussions/650
